src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs

```diff
@@ -58,8 +58,6 @@ public IActionResult Index([FromRoute] int ukprn)
     [HttpPost]
     public async Task<IActionResult> Index([FromRoute] int ukprn, SearchByPayeSubmitModel submitModel, CancellationToken cancellationToken)
     {
-        submitModel.Paye = submitModel.Paye!.Trim();
-        submitModel.Aorn = submitModel.Aorn!.Trim();
         var result = _validator.Validate(submitModel);
```

Comment:
Looking at the PAYE and AORN regex in the validator, these will fail validation if the input has leading/trailing whitespace